dotenv/mod.ts

```diff
@@ -1,187 +1,107 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 
```

Comment:
```suggestion
// This module is browser compatible.

```

This should be marked if it's true. (I didn't test with the compiler, but it appears to be.)